lib/test/Assertion.js

```diff
@@ -214,6 +215,14 @@ class Assertions {
 
         // We allow for a wild-card *
         let regex = expectedValue.trim().split("*").join(".*");
+        if (this._lenientMode) {
+            const toRemoveRegex = /[\f\n\r\t\v.,;:!?]/g;
+            regex = regex.replace(toRemoveRegex, "");
+            regex = regex.replace(/  +/g, " ");
```

Comment:
I don't think we should alter the expected value other than the wildcards on line 217. If a user enables `lenientMode` and then puts an expected value with special chars, it would be the user's fault.